### PR TITLE
Ignore files based on globbing.

### PR DESCRIPTION
**Pull request by SevereOverfl0w**
Ignore files if they match a glob. As discussed in https://github.com/anandthakker/doiuse/issues/29

### CODE REVIEW

**Comment by SevereOverfl0w** [comment]
Hmm, test is failing. I imagine it's due to the glob not being inclusive enough in this case.

File: README.md [hunk 1]
```diff
@@ -117,6 +119,7 @@ Yields `UsageInfo` objects as described below.
 {
   browsers: ['ie >= 8', '> 1%'] // an autoprefixer-like array of browsers.
   ignore: ['rem'], // an optional array of features to ignore
+  ignoreFiles: ['**/normalize.css'], // an optional array of file globs to ignore
```
Comment (anandthakker) [inline README.md]: Maybe add a note that these globs refer to the original source file path, using the source map?

**Comment by anandthakker** [comment]
@SevereOverfl0w any luck getting the test to pass?

**Comment by SevereOverfl0w** [comment]
Looks like I messed something up in the tests with my flattening of promises, I'm taking a look now.

**Comment by SevereOverfl0w** [comment]
Test now is passing!

However, I've just noticed something that I overlooked previously. I only noticed due to your comment on the docs, the ignore can be either absolute (essentially it's `root` + `from`) or relative (just `from`). I'm going to assume that `from` is the best to use, as I doubt you'd want to be polluted by the full path when ignoring files (`/home/user/code/projects/something/else/src/css/normalize.css`).

**Comment by anandthakker** [comment]
@SevereOverfl0w Nice!

Now that I think about it, seems like it would be good to have a test of your original use case in here, too, using `postcss-import`

**Comment by SevereOverfl0w** [comment]
Just reread what `input.from` is, it's absolute, not relative. But it is definitely the correct one to use. Doh!

Either way, that should be a test case for postcss-import, I based it on your feature ignore test.

**Comment by anandthakker** [comment]
Nice, this looks great.  Thanks for the follow-through, @SevereOverfl0w.

**Comment by anandthakker** [comment]
I'm also going to add you as a collaborator, as per [OPEN open source](https://github.com/anandthakker/doiuse/blob/master/CONTRIBUTING.md)

**Pull request merged**